openshift_tooling/prometheus_db_dump/prometheus_dump.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+output_dir=$1
+prometheus_namespace=openshift-monitoring
+
+if [[ "$#" -ne 1 ]]; then
+	 echo "syntax: $0 <output_dir>"
+	 exit 1
+fi
+
+# Check for kubeconfig
+if [[ ! -s $HOME/.kube/config ]]; then
+        echo "cannot find kube config in the home directory, please check"
+        exit 1
+fi
+
+# Check if oc client is installed
+which oc &>/dev/null
+echo "Checking if oc client is installed"
+if [[ $? != 0 ]]; then
+        echo "oc client is not installed, please check"
+        exit 1
+fi
+
+# prometheus pods
+prometheus_pod=$(oc get pods -n $prometheus_namespace | grep "prometheus-k8s-*" | awk -F " " '{print $1}' | tail -n1)
+
+# copy the prometheus DB from the prometheus pod
+oc cp $prometheus_namespace/$prometheus_pod:/prometheus/wal wal/
```

Comment:
Sucks that RHCOS doesn't ship with rsync because that would make this work. `cp` doesn't seem to copy the folder itself just the contents. So no `wal` directory is created, just the files, which means the `tar` always fail as well. Just to be explicit maybe we want to add `-c prometheus`?